sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1131,6 +1121,63 @@ export interface SearchDocumentsResultBase {
     };
 }
 
+// @public
+export class SearchIndexClient {
+    constructor(endpoint: string, credential: KeyCredential, options?: SearchIndexClientOptions);
+    analyzeText(indexName: string, options: AnalyzeTextOptions): Promise<AnalyzeResult>;
+    readonly apiVersion: string;
+    createIndex(index: Index, options?: CreateIndexOptions): Promise<Index>;
+    createOrUpdateIndex(index: Index, options?: CreateOrUpdateIndexOptions): Promise<Index>;
+    createOrUpdateSynonymMap(synonymMap: SynonymMap, options?: CreateOrUpdateSynonymMapOptions): Promise<SynonymMap>;
+    createSynonymMap(synonymMap: SynonymMap, options?: CreateSynonymMapOptions): Promise<SynonymMap>;
+    deleteIndex(index: string | Index, options?: DeleteIndexOptions): Promise<void>;
+    deleteSynonymMap(synonymMap: string | SynonymMap, options?: DeleteSynonymMapOptions): Promise<void>;
+    readonly endpoint: string;
+    getIndex(indexName: string, options?: GetIndexOptions): Promise<Index>;
+    getIndexStatistics(indexName: string, options?: GetIndexStatisticsOptions): Promise<GetIndexStatisticsResult>;
+    getSearchClient<T>(indexName: string, options?: SearchClientOptions): SearchClient<T>;
+    getServiceStatistics(options?: GetServiceStatisticsOptions): Promise<ServiceStatistics>;
+    getSynonymMap(synonymMapName: string, options?: GetSynonymMapsOptions): Promise<SynonymMap>;
+    listIndexes<Fields extends keyof Index>(options?: ListIndexesOptions): Promise<Array<Pick<Index, Fields>>>;
```

Comment:
we can remove the generic on this method now that select is removed, right?

Reply:
Done